modules/ckanext-ytp_main/ckanext/ytp/plugin.py

```diff
@@ -431,12 +431,17 @@ def setup_template_variables(self, context, data_dict):
                 raise
 
     # IFacets #
+    _LOCALE_ALIASES = {'en_GB': 'en'}
 
     def dataset_facets(self, facets_dict, package_type):
         facets_dict = OrderedDict()
+        lang = i18n.get_lang()
+        if lang in self._LOCALE_ALIASES:
+            lang = self._LOCALE_ALIASES[lang]
+
```

Comment:
This could use the existing helper https://github.com/vrk-kpa/opendata/blob/master/modules/ckanext-ytp_main/ckanext/ytp/helpers.py#L433 which already is imported to the file.